openpype/tools/project_manager/project_manager/window.py

```diff
@@ -15,8 +16,11 @@
 from openpype.widgets import PasswordDialog
 
 from openpype import resources
+from openpype.api import get_project_basic_paths, create_project_folders
 from avalon.api import AvalonMongoDB
 
+log = logging.getLogger(__name__)
```

Comment:
If it's because of logging in ProjectManagerWindow then create log insidie it. Use rather openpype logger:
```
from openpype.api import Logger

log = Logger.get_logger("ProjectManagerWindow")
```

Reply:
modified as suggested.